app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -68,6 +67,11 @@ public function index(Request $request)
                 $query = $query->where('signups.northstar_id', '=', $filters['northstar_id']);
             }
 
+            // Only return Posts for the given status (if there is one)
+            if (array_has($filters, 'status')) {
+                $query = $query->where('status', '=', $filters['status']);
```

Comment:
Do we need to be able to filter by more than one status here? e.g. `status=approved,pending`?